src/Domain/Creators/TimeDependence/RegisterDerivedWithCharm.cpp

```diff
@@ -26,9 +25,12 @@ struct get_maps {
 
 template <size_t Dim>
 void register_maps_with_charm() noexcept {
-  using maps_to_register = tmpl::remove_duplicates<tmpl::flatten<
-      tmpl::transform<typename TimeDependence<Dim>::creatable_classes,
-                      get_maps<tmpl::_1>>>>;
+  using maps_to_register =
+      tmpl::remove_duplicates<tmpl::flatten<tmpl::push_back<
+          tmpl::transform<typename TimeDependence<Dim>::creatable_classes,
+                          get_maps<tmpl::_1>>,
+          domain::CoordinateMap<Frame::Grid, Frame::Inertial,
```

Comment:
I think this addition is causing the clang-9, Debug, ON build failure, (which is built with shared libs) because it requires this file to need an instantiation of `domain::CoordinateMap<...>` which requires the include of `CoordinateMap.tpp`